src/assert.js

```diff
@@ -119,19 +119,17 @@ Assert.prototype = {
 	},
 
 	equal: function( actual, expected, message ) {
-		/*jshint eqeqeq:false */
 		this.pushResult( {
-			result: expected == actual,
+			result: expected == actual,		// eslint-disable-line eqeqeq
```

Comment:
We historically use the comment in the line above after a blank line. So I'm biased towards that as I am used to it. There are other in-comments rules placed like this, so I'm again leaning towards this way.

The second option looks like the best, but I'm open to discuss.